src/hooks/useFocusTrap.ts

```diff
@@ -28,25 +28,30 @@ interface FocusTrapHookSettings {
  * @param settings {FocusTrapHookSettings}
  */
 export function useFocusTrap(
-  settings?: FocusTrapHookSettings
+  settings?: FocusTrapHookSettings,
+  dependencies?: React.DependencyList
```

Comment:
Like the `useAnchoredPosition` change above, maybe do the same here?

```suggestion
  dependencies: React.DependencyList = []
```